### PR TITLE
Blog overview page

### PR DESCRIPTION
**Pull request by mikkelcodes**

### CODE REVIEW

**Comment by mikkelcodes** [comment]
Probably a few commits here and there that could be squashed

File: packages/website/gatsby-config.js [hunk 1]
```diff
@@ -18,6 +18,13 @@ module.exports = {
         overlayDrafts: true,
       },
     },
+    {
+      resolve: `gatsby-source-filesystem`,
```
Comment (TheKnarf) [inline packages/website/gatsby-config.js:22]: What is this needed for?

**Comment by TheKnarf** [comment]
Build error:

```
Can't resolve '../blogTagbar' in '/home/runner/work/alv-website/alv-website/packages/website/src/components/blogSection'
```

The import is importing `blogTagbar` but the folder is called `blogTagBar`, I recommend renaming it to `blogTagbar`.

File: packages/website/src/components/tagbar/index.js [hunk 2]
```diff
@@ -1,12 +1,13 @@
 import React, { useState } from 'react';
 import * as Icon from '../icon';
-import { EmployeesContainer } from '../employeesContainer';
+import { GridContainer } from '../gridContainer';
 import { Checkbox } from '../checkbox';
 
 export const Tagbar = ({ sanityTags, sanityEmployees }) => {
-  const [filteredEmployees, setFilteredEmployees] = useState(
+  const [filteredContent, setFilteredContent] = useState(
```
Comment (TheKnarf) [inline packages/website/src/components/tagbar/index.js:7]: Not sure why this was renamed? I think `filteredEmployees` made more sense here than just content?

File: packages/website/src/components/tagbar/index.js [hunk 3]
```diff
@@ -63,7 +64,7 @@ export const Tagbar = ({ sanityTags, sanityEmployees }) => {
         </div>
       </div>
       {/* CONTAINER WITH CARDS */}
-      <EmployeesContainer filteredEmployees={filteredEmployees} />
+      <GridContainer filteredContent={filteredContent} />
```
Comment (TheKnarf) [inline packages/website/src/components/tagbar/index.js:67]: Are we reusing the GridContainer other places?

File: packages/website/gatsby-config.js (hunk 1 above)
Comment (mikkelcodes) [reply packages/website/gatsby-config.js:22]: This is used to source a fluid fallback image for whenever we need fallback images

Comment (TheKnarf) [reply packages/website/gatsby-config.js:22]: I'm confused, didn't that already work without that plugin?

File: packages/website/src/components/tagbar/index.js (hunk 2 above)
Comment (mikkelcodes) [reply packages/website/src/components/tagbar/index.js:7]: The idea was that the same component will be used for filtering blog posts as well, so I changed to a more generic name

File: packages/website/src/components/tagbar/index.js (hunk 3 above)
Comment (mikkelcodes) [reply packages/website/src/components/tagbar/index.js:67]: Yes, the idea behind this change is the same as the above. Reusing the container

File: packages/website/gatsby-config.js (hunk 1 above)
Comment (mikkelcodes) [reply packages/website/gatsby-config.js:22]: Yes exactly, this is for use with gatsby-image

**Comment by mikkelcodes** [comment]
> Build error:
> 
> ```
> Can't resolve '../blogTagbar' in '/home/runner/work/alv-website/alv-website/packages/website/src/components/blogSection'
> ```
> 
> The import is importing `blogTagbar` but the folder is called `blogTagBar`, I recommend renaming it to `blogTagbar`.

Good recommendation

**Review by TheKnarf — APPROVED** [review]

**Pull request merged**